### PR TITLE
CAL-281 added application reference section to templated docs.

### PR DESCRIPTION
**Pull request by ricklarsen**
#### What does this PR do?

updates documentation templates to add application reference section to the appendix.

#### Who is reviewing it? 
@mcalcote @vinamartin @emmberk @Lambeaux @jrnorth @ahoffer 

#### Choose 2 committers to review/merge the PR.

@clockard
@coyotesqrl
@lessarderic
@pklinef
@shaundmorris
@stustison
@ricklarsen - Documentation
#### How should this be tested?

Check that all relevant sections are being included. "New" .adoc files are existing content with header information added. Content review is *OPTIONAL*.

#### What are the relevant tickets?

[CAL-281](https://codice.atlassian.net/browse/CAL-281)
[DDF-2891](https://codice.atlassian.net/browse/DDF-2891)

#### Screenshots (if appropriate)

[jdocumentation.pdf](https://github.com/codice/alliance/files/1212280/jdocumentation.pdf)

#### Checklist:
- [x] Documentation Updated
- [ ] Update / Add Unit Tests
- [ ] Update / Add Integration Tests

### CODE REVIEW

File: distribution/docs/src/main/jdocs/content/_plugins/nitf-render-plugin.adoc [hunk 1]
```diff
@@ -26,5 +26,6 @@ To configure the NITF Render Plugin:
 
 These configurations are available:
```
Comment (emmberk) [inline distribution/docs/src/main/jdocs/content/_plugins/nitf-render-plugin.adoc]: This line can probably be removed.

File: distribution/docs/src/main/jdocs/content/_tables/org.codice.alliance.nsili.endpoint-table-contents.adoc [hunk 2]
```diff
@@ -1,5 +1,9 @@
-:type: documentation
+:title: NSILI Endpoint Configuration
+:id: org.codice.alliance.nsili.endpoint
+:type: table
 :status: published
+:application: ${alliance-nsili}
+:summary: NSILI Endpoint configuration.
```
Comment (emmberk) [inline distribution/docs/src/main/jdocs/content/_tables/org.codice.alliance.nsili.endpoint-table-contents.adoc]: configuration**s**.

File: distribution/docs/src/main/jdocs/content/_transformers/mpegts-input-transformer-contents.adoc [hunk 3]
```diff
@@ -28,6 +28,6 @@ To configure the MPEGTS Input Transformer:
 
 These are the configurable properties:
 
-include::{adoc-include-cal}/_tables/org.codice.alliance.transformer.video.MpegTsInputTransformer-table-contents.adoc[]
+See <<org.codice.alliance.transformer.video.MpegTsInputTransformer,.MPEGTS Input Transformer configurations>> for all possible configurations.
```
Comment (emmberk) [inline distribution/docs/src/main/jdocs/content/_transformers/mpegts-input-transformer-contents.adoc]: Is there an unnecessary `.` before `MPEGTS` in this line?

File: distribution/docs/src/main/jdocs/content/_plugins/nitf-render-plugin.adoc (hunk 1 above)
Comment (ricklarsen) [inline distribution/docs/src/main/jdocs/content/_plugins/nitf-render-plugin.adoc]: done.

File: distribution/docs/src/main/jdocs/content/_tables/org.codice.alliance.nsili.endpoint-table-contents.adoc (hunk 2 above)
Comment (ricklarsen) [inline distribution/docs/src/main/jdocs/content/_tables/org.codice.alliance.nsili.endpoint-table-contents.adoc]: fixed.

File: distribution/docs/src/main/jdocs/content/_transformers/mpegts-input-transformer-contents.adoc (hunk 3 above)
Comment (ricklarsen) [inline distribution/docs/src/main/jdocs/content/_transformers/mpegts-input-transformer-contents.adoc]: yes. Removed.

File: distribution/docs/src/main/jdocs/content/_transformers/mpegts-input-transformer-contents.adoc [hunk 4]
```diff
@@ -7,7 +7,7 @@
 
 The MPEGTS Input Transformer extracts STANAG-4609 metadata from MPEG-TS files.
 
-===== Installing the MPEGTS Input Transformer
+===== Installing the org.codice.alliance.transformer.video.MpegTsInputTransformer
```
Comment (clockard) [inline distribution/docs/src/main/jdocs/content/_transformers/mpegts-input-transformer-contents.adoc]: Do we really want the package name in this title?

File: distribution/docs/src/main/jdocs/content/_applicationReferences/managing-nsili-contents.adoc [hunk 5]
```diff
@@ -0,0 +1,26 @@
+:title: ${alliance-nsili}
+:status: published
+:type: applicationReference
+:summary: Provides a bidirectional interface to STANAG-4559 Edition 3 compliant systems.
+:order: 22
+
+The *${alliance-nsili} Application* provides a bidirectional interface to STANAG-4559 Edition 3 compliant systems.
```
Comment (vinamartin) [inline distribution/docs/src/main/jdocs/content/_applicationReferences/managing-nsili-contents.adoc]: ❓  ✏️ Why is this one bold when the other ones aren't?

Comment (ricklarsen) [inline distribution/docs/src/main/jdocs/content/_applicationReferences/managing-nsili-contents.adoc]: fixed.

File: distribution/docs/src/main/jdocs/content/_transformers/mpegts-input-transformer-contents.adoc (hunk 4 above)
Comment (ricklarsen) [inline distribution/docs/src/main/jdocs/content/_transformers/mpegts-input-transformer-contents.adoc]: Nope. typo. fixed.

**Comment by ricklarsen** [comment]
build now

**Pull request merged**